AEGIS.Tests.Core.Reference/Operations/Projections/LambertAzimuthalEqualAreaProjectionTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿/// <copyright file="LabordeObliqueMercatorProjectionTest.cs" company="Eötvös Loránd University (ELTE)">
+///     Copyright (c) 2011-2022 Roberto Giachetta. Licensed under the
+///     Educational Community License, Version 2.0 (the "License"); you may
+///     not use this file except in compliance with the License. You may
+///     obtain a copy of the License at
+///     http://opensource.org/licenses/ECL-2.0
+///
+///     Unless required by applicable law or agreed to in writing,
+///     software distributed under the License is distributed on an "AS IS"
+///     BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+///     or implied. See the License for the specific language governing
+///     permissions and limitations under the License.
+/// </copyright>
+/// <author>Roberto Giachetta</author>
+
+using ELTE.AEGIS;
+using ELTE.AEGIS.Reference;
+using ELTE.AEGIS.Reference.Operations;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace AEGIS.Tests.Reference.Operations.Projections
+{
+    [TestFixture]
+    public class LambertAzimuthalEqualAreaProjectionTest
+    {
+        private LambertAzimuthalEqualAreaProjection _projection;
+        private Coordinate coordinate;
+        private GeoCoordinate geoCoordinate;
+
+        public LambertAzimuthalEqualAreaProjectionTest()
+        {
+            // Location: Berlin
+            coordinate = new Coordinate(4553442.949403, 3271908.232428);
+            geoCoordinate = new GeoCoordinate(52.5072003, 13.4247528);
```

Comment:
```suggestion
            geoCoordinate = new GeoCoordinate(Angle.FromDegree(52.5072003), Angle.FromDegree(13.4247528));
```

Otherwise `geoCoordinate` will be *invalid*, as it expects radian parameters.